content/docs/reference/pkg/_index.md

```diff
@@ -79,6 +80,7 @@ Each cloud vendor has a dedicated package for deploying resources to it:
 * [**Kubernetes** (`pulumi_kubernetes`)]({{< relref "python/pulumi_kubernetes" >}})
 
 * [**Cloudflare** (`pulumi_cloudflare`)]({{< relref "python/pulumi_cloudflare" >}})
+* [**HashiCorp Consul** (`pulumi_consul`)]({{< relref "python/pulumi_consul" >}})
```

Comment:
Why is it just "Consul" for Node, but "HashiCorp Consul" for Python? Pick one and use for both.

Reply:
I will update that in this PR I am sending now! Good spot